server/lib/collectivelib.ts

```diff
@@ -148,11 +149,10 @@ export function validateSettings(settings: any): string | boolean {
   }
 
   /*
-   * Validate customEmailMessage length. We are using 511 characters to account
-   * for the "div" tags that enclose the content <div>..<div/>. These divs are auto
-   * added by the RichTextEditor.js
+   * Validate customEmailMessage length.
    */
-  if (settings.customEmailMessage && settings.customEmailMessage.length > 511) {
+  if (settings.customEmailMessage && stripHTML(settings.customEmailMessage).length > 500) {
+    console.log(stripHTML(settings.customEmailMessage));
     return 'Message should be less than 500 characters';
```

Comment:
Would be great to be more explicit on this one.

```suggestion
    return 'Custom "Thank you" email message should be less than 500 characters';
```